aten/src/ATen/native/NaiveConvolutionTranspose3d.cpp

```diff
@@ -311,7 +295,7 @@ void slow_conv_transpose3d_out_cpu_template(
           // (see http://docs.nvidia.com/cuda/cublas/#cublas-lt-t-gt-gemm)
           const int64_t m =
               weight.size(1) * weight.size(2) * weight.size(3) * weight.size(4);
-          const int64_t n = columns.size(1);
+          const int64_t n = input_depth * input_height * input_width;
```

Comment:
I don't strictly need to change it - this was just for consistency with the backward. Since columns is unconditionally being allocated above, I'll change it back